package.json

```diff
@@ -101,11 +101,11 @@
     "marked": "^0.7.0",
     "moment": "^2.17.1",
     "netlify": "^1.2.0",
-    "prop-types": "^15.5.10",
+    "prop-types": "^15.7.2",
     "pvutils": "^1.0.17",
-    "react": "^16.8.4",
+    "react": "^16.14.0",
```

Comment:
The latest version of the components [has been updated to use React v17](https://github.com/webex/components/blob/master/package.json#L38)